### PR TITLE
Config change to enable map voting

### PR DESCRIPTION
**Pull request by Rukofamicom**
## About The Pull Request
Enables map voting
Adds a config option to prevent map vote if a certain round length hasn't been reached. 

Fixes minor oversight in #6776 - I only set it to force a rotation when reading from map preference, not when creating the map vote. Map will now *actually* always rotate. 

## Why It's Good For The Game
Should have been part of #6776. Updating map voting without enabling it as something that happens was silly

## Testing Photographs and Procedure

It is tested, and this time by me instead of an outside person. 
This is how I caught the current map going up for vote when it shouldn't have been. 

## Changelog
:cl:
config: Map voting is enabled again
config: Map votes may now have a minimum round length defined before they trigger
fix: Map votes will always exclude the current map from the voting pool. 
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Problem is it will end up being something like Meta > Delta > meta > Delta > meta > delta > meta > box > meta > delta

**Comment by Mat05usz** [comment]
> Problem is it will end up being something like Meta > Delta > meta > Delta > meta > delta > meta > box > meta > delta

Votes were changed to be weighted instead, I think it would be nice to at least test it to see how it goes. It's better than map X being played 8th or so times in a row.

**Comment by Rukofamicom** [comment]
> Problem is it will end up being something like Meta > Delta > meta > Delta > meta > delta > meta > box > meta > delta

Meta and delta are definitely popular maps, but I don't think they're still as overwhelmingly popular as you do. Even if they are, it's not necessarily great to frequently force players into maps they don't like just because the map exists. 

I can also just add some additional weighting to the votes later if it really is necessary: Add 3 to each possible option in the poll so there is always a minimum chance for each map.

File: config/config.txt [hunk 1]
```diff
@@ -431,7 +431,11 @@ MAPROTATION
 PREFERENCE_MAP_VOTING 0
 
 ## Starts a map vote at the end of rounds
-#AUTOMAPVOTE
+AUTOMAPVOTE
+
+## Sets a minimum round length (in minutes) before a map vote will be called, to discourage players from forcing early round ends just to change map
```
Comment (Crossedfall) [inline config/config.txt]:
```suggestion
## Sets a minimum round length (in minutes) before a map vote will be called to discourage players from forcing early round ends just to change map
```

**Review by Crossedfall — COMMENTED** [review]
I'm indifferent to this change overall. We moved away from map votes because it created a constant loop of the same few maps and people got really tired of it. Enough time has passed to where we can probably try it again, but I imagine this'll end up being a repeat of our past.

**Comment by Rukofamicom** [comment]
> but I imagine this'll end up being a repeat of our past.

If this is still an issue with draw-from-a-hat style voting let me know and I can provide a system to tip the scales toward including more instances of the lesser loved maps without forcing them as frequently as full RNG. 

var/x = round(players online / 10)
Drawing votes for map:
* Meta: 18 +x
* Delta: 10 +x
* Corg: 2 +x
* Kilo: 3 +x

**Review by Crossedfall — APPROVED** [review]
No countering opinions. Going to merge and enable in the config. Worst case we revert back to rotations.

**Pull request merged**